packs/data/bestiary-effects.db/effect-despair.json

```diff
@@ -0,0 +1,48 @@
+{
+    "_id": "1bOSJ2LbEC28aI9f",
+    "img": "systems/pf2e/icons/spells/canticle-of-everlasting-grief.webp",
+    "name": "Effect: Despair",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Living creatures are frightened 1 while in a despair aura. They can't naturally recover from this fear while in the area but recover instantly once they leave the area.</p>"
```

Comment:
Don't know how I feel about this one, it's just a container for frightened 1. May be worth waiting until the aura can predicate who it grants to instead.

Reply:
Containers for conditions are important because it's how traits and other information is transmitted.